streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -353,8 +349,7 @@ public void shouldThrowOnUnassignedStateStoreAccess() {
             .addProcessor(badNodeName, new LocalMockProcessorSupplier(), sourceNodeName);
 
         try {
-            new TopologyTestDriver(topology, mkProperties(
-                mkMap(mkEntry(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory().getAbsolutePath()))));
+            new TopologyTestDriver(topology);
```

Comment:
Is it ok to remove ```StreamsConfig.STATE_DIR_CONFIG``` here?

Reply:
That's a good point. This test is validating if exception is thrown when topology is created with processor node which doesn't have state store specified. Providing STATE_DIR_CONFIG config parameter in properties won't alter desired outcome of the test. Hence I removed it.